tests/test_modeling_distilbert.py

```diff
@@ -261,7 +266,7 @@ def test_torchscript_device_change(self):
 
             # BertForMultipleChoice behaves incorrectly in JIT environments.
```

Comment:
Do we want to skip JIT testing for `DistilBertForQuestionAnswering`, `DistilBertForSequenceClassification`, and `DistilBertForTokenClassification` as well (these classes come after `BertForMultipleChoice`).